.drone/README.md

```diff
@@ -18,3 +22,21 @@ Alternatively, run the following `make` command from the project root directory:
 ```
 make drone
```

Comment:
if we have a Make command that does things automatically, I see no need to document the individual steps.  If someone wants to know how it works, they can look at the makefile.

Reply:
Will do!